### PR TITLE
List docker check changes in the documentation.

### PR DESCRIPTION
**Pull request by hush-hush**
### What does this PR do?

List docker check changes in the documentation.

### CODE REVIEW

File: docs/beta/changes.md [hunk 1]
```diff
@@ -106,6 +106,40 @@ to:
...
+The new check is named `docker` and no longer `docker_daemon`.
+
+For now we support a subset of metrics, docker events and `docker.status`
+service check. Look into
+[`docker.yaml.example`](/pkg/collector/dist/conf.d/docker.yaml.example) for
+more information.
+
+Main changes:
+
+- Some option has been moved from `docker_daemon.yaml` to the main `datadog.yaml`.
```
Comment (hkaj) [inline docs/beta/changes.md]: rather `some options have`? If there's just one we can just name it here.

**Review by hkaj — CHANGES_REQUESTED** [review]

File: docs/beta/changes.md (hunk 1 above)
Comment (hush-hush) [reply docs/beta/changes.md]: The next block is the list of options that have moved

File: docs/beta/changes.md [hunk 2]
```diff
@@ -106,6 +106,40 @@ to:
 gauge(self, name, value, tags=None, hostname=None, device_name=None)
 ```
 
+### Docker check
+
+The Docker checks is being rewritten in Go to take advantage of the new
```
Comment (hkaj) [inline docs/beta/changes.md]: checks --> check

File: docs/beta/changes.md [hunk 3]
```diff
@@ -106,6 +106,40 @@ to:
...
+internal architecture of the Agent, mainly bringing a consistent behaviour
+across every container related component. Therefore the Python versions will
```
Comment (hkaj) [inline docs/beta/changes.md]: versions --> version

File: docs/beta/changes.md [hunk 4]
```diff
@@ -106,6 +106,40 @@ to:
...
+The new check is named `docker` and no longer `docker_daemon`.
+
+For now we support a subset of metrics, docker events and `docker.status`
+service check. Look into
+[`docker.yaml.example`](/pkg/collector/dist/conf.d/docker.yaml.example) for
+more information.
+
+Main changes:
+
+- Some option have moved from `docker_daemon.yaml` to the main `datadog.yaml`:
```
Comment (hkaj) [inline docs/beta/changes.md]: option --> options

File: docs/beta/changes.md [hunk 5]
```diff
@@ -106,6 +106,40 @@ to:
...
+  * `docker_root` option has been split in two options `container_cgroup_root`
+    and `container_proc_root`.
+  * `exclude` and `include` list has been renamed `ac_include` and
```
Comment (hkaj) [inline docs/beta/changes.md]: list has --> lists have

File: docs/beta/changes.md [hunk 6]
```diff
@@ -106,6 +106,40 @@ to:
...
+    `ac_exclude`. They will impact every container related component of the
```
Comment (hkaj) [inline docs/beta/changes.md]: container related --> container-related

File: docs/beta/changes.md [hunk 7]
```diff
@@ -106,6 +106,40 @@ to:
...
+    agent. Those lists that only supports image name and container name
```
Comment (hkaj) [inline docs/beta/changes.md]: Those lists that supports --> Those lists support

File: docs/beta/changes.md [hunk 8]
```diff
@@ -106,6 +106,40 @@ to:
...
+    (instead of any tags).
+  * `exclude_pause_container` has been added to exclude pause containers on
+    Kubernetes (default to true). This will avoid users removing them from the
```
Comment (hkaj) [inline docs/beta/changes.md]: Kubernetes and Openshift

File: docs/beta/changes.md [hunk 9]
```diff
@@ -106,6 +106,40 @@ to:
...
+    exclude list by error.
+  * `collect_labels_as_tags` has been renamed `docker_labels_as_tags` and now
+    support high cardinality tags
```
Comment (hkaj) [inline docs/beta/changes.md]: support --> supports

File: docs/beta/changes.md [hunk 10]
```diff
@@ -106,6 +106,40 @@ to:
...
+
+The [`import`](#configuration-files) command support a `--docker` flag to convert the old
```
Comment (hkaj) [inline docs/beta/changes.md]: support --> supports

File: docs/beta/changes.md [hunk 11]
```diff
@@ -106,6 +106,40 @@ to:
...
+`docker_daemon.yaml` to the new `docker.yaml`. The command will also move
+needed setting from `docker_daemon.yaml` to `datadog.yaml`.
```
Comment (hkaj) [inline docs/beta/changes.md]: setting --> settings

**Review by hkaj — CHANGES_REQUESTED** [review]

**Review by hkaj — APPROVED** [review]

**Pull request merged**